src/main/java/org/commcare/formplayer/beans/menus/PeristentCommand.java

```diff
@@ -44,4 +58,20 @@ public ArrayList<PeristentCommand> getCommands() {
     public void addCommand(PeristentCommand command) {
         commands.add(command);
     }
+
+    public NavIconState getNavigationState() {
+        return navigationState;
```

Comment:
does this get used anywhere on Web Apps for menu screens ?

Reply:
Yes, it's used by web apps [here](https://github.com/dimagi/commcare-hq/blob/96da31d98ba42fadc12c8b2568652a7777a19209/corehq/apps/cloudcare/templates/cloudcare/partials/menu/row.html#L9-L13) to determine which default icon to use (the pencil for forms or folder for menus)

Reply:
Ahh that's super good to know, think it's worth to add a comment somewhere for this to describe what it does as the naming for this is quite obscure to know what it's used for

Reply:
[7aed2c5](https://github.com/dimagi/formplayer/pull/1603/commits/7aed2c59030a1920350acfd46f03537dfadac393)